### PR TITLE
Civic Liker V2 UI

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: src/components/CivicLikerWelcome/CivicLikerWelcomeView.vue [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  name: 'CivicLikerWelcomeView',
+  components: {
+    Identity,
+  },
+  props: {
+    price: {
+      type: Number,
+      default: 1,
+    },
+    referrerDisplayName: {
```
Comment (williamchong) [inline src/components/CivicLikerWelcome/CivicLikerWelcomeView.vue:106]: Does there exists a case where we would want to use this component without a referrer? Like when it falls back to `foundation`

File: src/pages/settings/support.vue [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+      await Promise.all([
+        this.fetchCivicSupportingUsers(),
+        this.fetchUserSubscriptionInfo(),
+      ]);
+    } catch (err) {
+      // eslint-disable-next-line no-console
+      console.error(err);
+      this.state = 'error-civic-v2';
+    }
+    this.state = 'done';
```
Comment (williamchong) [inline src/pages/settings/support.vue:162]: Does this make `error-civic-v2` always get replaced by `done`?

File: src/pages/settings/support/users/_id.vue [hunk 3]
```diff
@@ -97,36 +319,101 @@ export default {
...
+    confirmQuantity() {
+      this.state = this.getUserIsCivicLiker ? 'confirm' : 'new';
+    },
+
+    newSubscription() {
+      this.$router.push({
+        name: `civic-register-stripe`,
+        query: {
+          from: this.authorId,
+          civic_liker_version: 2,
```
Comment (williamchong) [inline src/pages/settings/support/users/_id.vue:367]: We would remove `civic_liker_version` to default everyone to `2` when public release

**Review by williamchong — CHANGES_REQUESTED** [review]
Do we know what would happen for `/civic` when v2 release yet? Seems big part of `settings/support/users/index.vue` maybe be useful in `/civic` for on board, so maybe need to refactor in future(not sure).

File: src/pages/settings/support.vue (hunk 2 above)
Comment (nwingt) [reply src/pages/settings/support.vue:162]: I blame you

File: src/pages/settings/support/users/_id.vue (hunk 3 above)
Comment (nwingt) [reply src/pages/settings/support/users/_id.vue:367]: Then, will remove this query after release

File: src/components/CivicLikerWelcome/CivicLikerWelcomeView.vue (hunk 1 above)
Comment (nwingt) [reply src/components/CivicLikerWelcome/CivicLikerWelcomeView.vue:107]: @joshkiu Do you have any comments?

**Review by williamchong — CHANGES_REQUESTED** [review]

File: src/components/CivicLikerPageContentV2.vue [hunk 4]
```diff
@@ -173,7 +173,7 @@
         )
           i18n.text-like-green.font-400(path="civicLiker" place="civicLiker")
 
-    +PaymentSection
+    //- +PaymentSection
```
[the remark this answers is not in the document]
Comment (nwingt) [reply src/components/CivicLikerPageContentV2.vue:176]: Will clean up at revamp

File: src/components/CivicLikerPageContentV2.vue [hunk 5]
```diff
@@ -241,7 +241,7 @@
           .civic-liker-page-v2__mobile-benefit-list-swiper-pagination(slot="pagination")
           .border-b.border-gray-9b.mx-16.mt-32(class="tablet:hidden")
 
-    +PaymentSection
+    //- +PaymentSection
```
[the remark this answers is not in the document]
Comment (nwingt) [reply src/components/CivicLikerPageContentV2.vue:244]: Will clean up at revamp

**Comment by nwingt** [comment]
Accidentally merged 😅

**Pull request merged**